spec.html

```diff
@@ -32989,13 +32987,11 @@
 
       <!-- es6num="24.1.2.1" -->
       <emu-clause id="sec-arraybuffer-length">
-        <h1>ArrayBuffer ( _length_ )</h1>
-        <p>ArrayBuffer called with argument _length_ performs the following steps:</p>
+        <h1>ArrayBuffer ( [ _length_ ] )</h1>
```

Comment:
I agree they are seldom called with no arguments, but it's more clear to have them being explicitly optional, otherwise I would seem like an error to call `new ArrayBuffer()`.